### DISCUSSION TITLE
CoreDisplay_DisplayCreateInfoDictionary(CGDirectDisplayID) function undefined

### DISCUSSION THREAD

**Discussion by coolcoolkg21** [OCJvanDijk/Brightness-Sync#39] [General]
Dear author,

I'm trying to build the project, but it fails and shows "Undefined symbol: _CoreDisplay_DisplayCreateInfoDictionary."
Do I have to import other header files or libraries to make this function work?
Thanks in advance.

**Comment by OCJvanDijk** [comment]
The Xcode project should just build without any additional steps. I just tried with a clean checkout using the latest Xcode and it still works for me...
